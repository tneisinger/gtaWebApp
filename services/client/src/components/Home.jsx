import React from 'react';

const Home = () => (
  <div>
    <h1>Home</h1>
    <hr/><br/>
    <p>Home goes here.</p>
  </div>
);

export default Home;
