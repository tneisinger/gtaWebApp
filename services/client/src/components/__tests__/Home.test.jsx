import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Home from '../Home';

test('Home renders properly', () => {
  const wrapper = shallow(<Home/>);
  const element = wrapper.find('p');
  expect(element.length).toBe(1);
  expect(element.text()).toBe('Home goes here.');
});

test('Home renders a snapshot properly', () => {
  const tree = renderer.create(<Home/>).toJSON();
  expect(tree).toMatchSnapshot();
});
