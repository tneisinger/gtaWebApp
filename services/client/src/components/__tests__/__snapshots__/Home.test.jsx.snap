// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a snapshot properly 1`] = `
<div>
  <h1>
    Home
  </h1>
  <hr />
  <br />
  <p>
    Home goes here.
  </p>
</div>
`;
